### ISSUE TITLE
`theme_tt("tabular")` with `modelsummary`

### ISSUE THREAD

**Issue by vincentarelbundock** [vincentarelbundock/tinytable#301]
The tabular theme is fantastic if you follow the steps on the website.

    If you use modelsummary, it doesn't work without some manual editing of the .tex output:

```
library(modelsummary)
library(tinytable)

reg<-lm(data=mtcars,mpg~cyl) 

# still has talltblr syntax after begin
modelsummary(reg)|> theme_tt('tabular') |> print('latex')

# still has talltblr syntax after begin
# ends with talltblr
modelsummary(reg, stars =  c('*' = .1, '**' = .05, '***' = 0.01))|> 
  theme_tt('tabular') |> print('latex')
```

_Originally posted by @mjdcurtis in https://github.com/vincentarelbundock/tinytable/issues/268#issuecomment-2212552257_

**Comment by vincentarelbundock** [comment]
https://github.com/vincentarelbundock/tinytable/issues/268#issuecomment-2212565953

**Comment by andrewheiss** [comment]
Ha, I just ran into this issue too when working with `theme_tt("tabular", style = "tabularray")`(for making a table cross-referencable in an appendix, via https://github.com/vincentarelbundock/tinytable/issues/293#issuecomment-2205794565)

The table in LaTeX starts with `\begin{tabular}` but ends with `\end{talltblr}`

**Comment by vincentarelbundock** [comment]
@mjdcurtis @andrewheiss 

I believe this is fixed in version 0.3.0.28 of `tinytable` from Github.
